libs/wire-api/src/Wire/API/Routes/Public/Galley/Conversation.hs

```diff
@@ -624,6 +624,18 @@ type ConversationAPI =
                :> ReqBody '[JSON] TypingData
                :> MultiVerb 'POST '[JSON] '[RespondEmpty 200 "Notification sent"] ()
            )
+    :<|> Named
+           "member-typing-qualified"
+           ( Summary "Sending typing notifications with domain"
```

Comment:
I would leave out the "with domain" part. This is the default (and starting with V3 the only) endpoint, so no need to say that it takes a qualified ID; we don't do that in other endpoint descriptions.